x/warp/config.go

```diff
@@ -197,12 +197,13 @@ func (c *Config) verifyPredicate(predicateContext *precompileconfig.PredicateCon
 	return c.verifyWarpMessage(predicateContext, warpMessage)
 }
 
-// VerifyPredicate verifies the predicate represents a valid signed and properly formatted Avalanche Warp Message.
+// VerifyPredicate computes indices of predicates that failed verification as a bitset then returns the result
+// as a byte slice.
 func (c *Config) VerifyPredicate(predicateContext *precompileconfig.PredicateContext, predicates [][]byte) []byte {
 	resultBitSet := set.NewBits()
 
 	for predicateIndex, predicateBytes := range predicates {
-		if c.verifyPredicate(predicateContext, predicateBytes) {
+		if !c.verifyPredicate(predicateContext, predicateBytes) {
```

Comment:
can we add a comment to explain this change here? a 1 is added to the bitset to describe failure instead of 0? or an empty bitset represents all successes?

Reply:
There's a comment for the function that explains this behavior, so I don't think it's useful to replicate it from a couple lines above here.